debezium-connector-mysql/pom.xml

```diff
@@ -74,6 +74,22 @@
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-avro-converter</artifactId>
         </dependency>
+        <!-- Used for unit testing with Docker -->
```

Comment:
Yes and no :-). I am keeping it there for two reasons

- The responsibility is now splitted, F8 builds, Cube runs. When we evaluate the typical development pattern we can either keep it or move image building to Cube too.
- It does explicit container cleanup after test suite execution. I am not sure if it is necessary to keep it but at the beginining I'd rather see container scleaned up twice instead of never ;-). Still the configuration can be cleaned to keep onyl build section and from runtime section only container naming.